snippets/go.json

```diff
@@ -156,6 +156,10 @@
 			"prefix": "tf",
 			"body": "func Test$1(t *testing.T) {\n\t$0\n}"
 		},
+		"benchmark function": {
+			"prefix": "bf",
+			"body": "func Benchmark$1(b *testing.B) {\n\tfor n := 0; n < b.N; n++ {\n\t\t$0\n\t}\n}"
```

Comment:
I will change `n` to `i`, but allowing to change that variable doesn't make sense for benchmark functions as Go chooses value of `b.N` on its own.